src/coreclr/src/vm/eeconfig.cpp

```diff
@@ -1451,3 +1451,9 @@ bool EEConfig::ShouldLogCCWRefCountChange(LPCUTF8 pszClassName, LPCUTF8 pszNames
     return false;
 }
 #endif // FEATURE_COMINTEROP
+ 
+int EEConfig::GetGCLargePages() const
```

Comment:
I think this can called directly in `GCToEEInterface::GetBooleanConfigValue`. It does not need to be here.